package.json

```diff
@@ -129,86 +129,88 @@
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
     "ent": "^2.2.0",
-    "filehound": "^1.17.0",
+    "filehound": "^1.17.3",
     "fs-extra": "^8.0.1",
     "gulp": "^4.0.1",
     "gulp-coffee": "^2.2.0",
     "gulp-inline-source": "^2.1.0",
     "gulp-shell": "^0.5.2",
-    "husky": "^3.0.9",
+    "husky": "^3.1.0",
     "intercept-stdout": "^0.1.2",
-    "mocha": "^6.2.0",
-    "nock": "^11.0.7",
+    "mocha": "^6.2.2",
+    "nock": "^11.7.2",
     "parse-link-header": "~1.0.1",
-    "pkg": "^4.4.0",
+    "pkg": "^4.4.2",
     "prettier": "^1.19.1",
-    "publish-release": "^1.6.0",
+    "publish-release": "^1.6.1",
     "resin-lint": "^3.1.1",
     "rewire": "^3.0.2",
-    "sinon": "^7.4.1",
-    "ts-node": "^8.1.0",
-    "typescript": "^3.4.5"
+    "sinon": "^7.5.0",
+    "ts-node": "^8.6.2",
+    "typescript": "^3.7.5"
   },
   "dependencies": {
-    "@oclif/command": "^1.5.12",
+    "@oclif/command": "^1.5.19",
     "@oclif/errors": "^1.2.2",
     "@resin.io/valid-email": "^0.1.0",
     "@zeit/dockerignore": "0.0.3",
     "JSONStream": "^1.0.3",
     "ansi-escapes": "^2.0.0",
     "any-promise": "^1.3.0",
     "archiver": "^2.1.0",
-    "balena-config-json": "^2.1.0",
-    "balena-device-init": "^5.0.0",
-    "balena-device-status": "^3.1.2",
-    "balena-image-manager": "^6.1.0",
-    "balena-preload": "^8.2.1",
-    "balena-sdk": "^12.16.0",
-    "balena-settings-client": "^4.0.0",
-    "balena-sync": "^10.0.3",
+    "balena-config-json": "^2.1.1",
+    "balena-device-init": "^5.0.1",
+    "balena-device-status": "^3.2.0",
+    "balena-errors": "^4.2.1",
```

Comment:
I added this explicit dependency because the cli does directly depend on it and so if dependencies change their balena-errors version it can break the cli (either because it's no longer available in node_modules, or the wrong version is available)